.github/workflows/ecli-image-aarch64.yml

```diff
@@ -0,0 +1,76 @@
+name: Build and public ecli docker image in non-x86
+
+on:
+  push:
+    branches: "master"
+    paths:
+      - "VERSION"
+
+jobs:
+  aarch64_job:
+    # The host should always be Linux
+    runs-on: ubuntu-22.04
+    name: Build on ubuntu-22.04 aarch64
+    steps:
+      - uses: actions/checkout@v3
+      - name: update remote
+        run: |
+          git submodule update --init --recursive --remote
+          mkdir ~/code
+          cp -r ../eunomia-bpf ~/code/
+
+      - uses: uraimo/run-on-arch-action@v2
+        name: Run commands
+        id: runcmd
+        with:
+          arch: aarch64
+          distro: ubuntu22.04
+          dockerRunArgs: |
+            -v ~/code/eunomia-bpf:/usr/eunomia-bpf
```

Comment:
Since all the files have been copied here, it may be possible to add rust-cache to this action.

But I don't know how to copy the cache out from docker, so this change is just a suggestion. If it can't be completed, forget it.

Reply:
Sorry, I didn't find the right way. . .